frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/entity/ReverificationResultRequest.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.di.authentication.frontendapi.entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import uk.gov.di.authentication.shared.entity.BaseFrontendRequest;
+import uk.gov.di.authentication.shared.validation.Required;
+
+public class ReverificationResultRequest extends BaseFrontendRequest {
```

Comment:
In general here we're moving away from having classes for request objects, and using records instead. It'll mean we need the right email serialisation.